### ISSUE TITLE
Add support for AAT's for the creation of data constants classifications

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-frontend#483]
**Problem Description:**
The labels for `caused_by` data is incorrect. 

**Expected Behavior/Solution:**
The labeling will be based on the AATs. The labels should be as follows:
```
Creation
                      Caused by Event:     Brewers
                      Additional Events:   Centennial
```

**Requirements:**

- [x] Update the function for getting the labels

Support for these AATs can be found in 
```
creation > part > classified_as:
Creator: 300386174 -> Caused By Event
Contributor: 300403974 -> Additional Events
```

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [x] Wireframe/Mockup - Kam
- [x] ~Committee discussions - Sarah~
- [x] ~Feasibility/Team discussion - Sarah~
- [x] ~Backend requirements - TBD~
- [x] Frontend requirements - Kam
- [ ] Are new regression tests required for QA - Amy
- [x] ~Questions~

**UAT/LUX Examples:**
- https://lux-front-sbx.collections.yale.edu/view/text/614f2504-a240-4720-8e2a-d87d9bf7eb5f 
- https://lux-front-sbx.collections.yale.edu/data/text/614f2504-a240-4720-8e2a-d87d9bf7eb5f (data of the above) 

**Dependencies/Blocks:**
N/A

**Related Github Issues:**
 https://git.yale.edu/LUX/unit-data/issues/208 

**Related links:**
N/A

**Wireframe/Mockup:**
<img width="913" alt="Image" src="https://github.com/user-attachments/assets/8cc073ef-fcf5-4eb1-aa12-f1ac5e1cf9ab" />

**Comment by roamye** [comment]
@kkdavis14  - I wanted to confirm that testing this is not possible until YUL provides these records, which is still to be determined. Is that correct?

**Comment by kkdavis14** [comment]
@roamye latest YUL data has this fixed: https://linked-art.library.yale.edu/node/008df9b6-860d-4135-aac9-7780ed048094

but the SBX record is not adding in the equivalent where it should:
https://lux-front-sbx.collections.yale.edu/data/text/614f2504-a240-4720-8e2a-d87d9bf7eb5f

This will have to go back to pipeline to see where the problem is. It should be getting added.

**Comment by roamye** [comment]
@kkdavis14 - thank you Kelly. Let us know when YUL gets the data into SBX so we can find UAT examples for this ticket.

**Comment by roamye** [comment]
@kkdavis14 - does the 01/10 data-set in SBX help us find the UAT examples?

**Comment by kkdavis14** [comment]
@roamye it does!
https://lux-front-sbx.collections.yale.edu/data/text/614f2504-a240-4720-8e2a-d87d9bf7eb5f

**Comment by kkdavis14** [comment]
@kamerynB Yes, it's not worded well in the description. I think this actually blocked https://github.com/project-lux/lux-frontend/issues/309 which is now closed, interestingly enough.

The correct AATs were needed for you to be able to pick them up in creation > part > caused_by > classified_as > equivalent > id, to render what Rob requested here:
https://github.com/project-lux/lux-frontend/issues/309#issuecomment-2305605267

**Comment by kamerynB** [comment]
@kkdavis14 Interesting. I think what's going on is that I added support to render those data points but with no support for the classified_as AATs for the agents. We can discuss and confirm next steps in forming today.

**Comment by zachjschwartz** [comment]
Deployed to prod. Closing.
